### PR TITLE
Adding a bit of doc

### PR DESCRIPTION
**Pull request by Grummfy**
Adding a minimal documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,40 @@ Supported Bridges:
...
+	$slackNotifier = $notifierFactory->make([
+		// specify that we will use slack
+		'driver'	=> 'slack',
+		// add api token to get access to slack API
+		'token'		=> ''
+	]);
+
+	/* @var \NotifyMeHQ\NotifyMe\Response $response */
+	$response =  $slackNotifier->notify('test message', [
+		'from'	=> 'Super Bot',
```
Comment (GrahamCampbell) [inline README.md]: Also, reduce the nesting level by one.

Comment (GrahamCampbell) [inline README.md]: And indent with tabs.

Comment (GrahamCampbell) [inline README.md]: And indent with spaces, not tabs please.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,40 @@ Supported Bridges:
...
+		'to'	=> '#sandbox'
```
Comment (GrahamCampbell) [inline README.md]: Missing comma.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,40 @@ Supported Bridges:
...
+* Notify : <code>$response = $notifier->notify($message, []);</code>
+* CHeck the response : <code>$response->isSent();</code>
+
+### Example
+Here is an exemple of a notification with Slack
+
+	<?php
+
+	// include autoloader
+	include __DIR__ . '/vendor/autoload.php';
```
Comment (GrahamCampbell) [inline README.md]: concatenate without spaces plz

File: README.md [hunk 4]
```diff
@@ -25,6 +25,40 @@ Supported Bridges:
...
+
+	// create a factory for notifications
+	$notifierFactory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();
+
+	// create the new notification for slack
+	$slackNotifier = $notifierFactory->make([
+		// specify that we will use slack
+		'driver'	=> 'slack',
```
Comment (GrahamCampbell) [inline README.md]: don't indent any more than required, and use spaces for alignment

**Comment by Grummfy** [comment]
I fix all your comment, I think.
Thanks

File: README.md [hunk 5]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
 * [Laravel 4](https://github.com/notifymehq/laravel4)
 * [Laravel 5](https://github.com/notifymehq/laravel5)
 
+## Usage
+* Create a factory : ```$factory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();```
```
Comment (GrahamCampbell) [inline README.md:30]: Missing new line.

File: README.md [hunk 6]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+* Make a notifier : ```$notifier = $factory->make([...]);```
+* Notify : ```$response = $notifier->notify($message, []);```
+* CHeck the response : ```$response->isSent();```
+
+### Example
+Here is an exemple of a notification with Slack
```
Comment (GrahamCampbell) [inline README.md:37]: Missing new line.

File: README.md [hunk 7]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
 * [Laravel 4](https://github.com/notifymehq/laravel4)
 * [Laravel 5](https://github.com/notifymehq/laravel5)
 
+## Usage
+* Create a factory : ```$factory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();```
+* Make a notifier : ```$notifier = $factory->make([...]);```
+* Notify : ```$response = $notifier->notify($message, []);```
+* CHeck the response : ```$response->isSent();```
```
Comment (GrahamCampbell) [inline README.md]: typo

Comment (GrahamCampbell) [inline README.md]: Also, please just use a single `.

File: README.md [hunk 8]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+$response =  $slackNotifier->notify('test message', [
+    // Who send this message, here is a bot called 'Super Bot'
+    'from'  => 'Super Bot',
+    // Set the channel to received the message
+    'to'    => '#sandbox'
+]);
+
+echo $response->isSent() ? 'Message sent' : 'Message going nowhere';
+```
+
```
Comment (GrahamCampbell) [inline README.md:66]: Too many new lines.

File: README.md [hunk 9]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+// include autoloader
+include __DIR__ . '/vendor/autoload.php';
+
+// create a factory for notifications
+$notifierFactory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();
+
+// create the new notification for slack
+$slackNotifier = $notifierFactory->make([
+    // specify that we will use slack
+    'driver'    => 'slack',
```
Comment (GrahamCampbell) [inline README.md]: Too many spaces.

File: README.md [hunk 10]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+    // add api token to get access to slack API
+    'token'     => ''
```
Comment (GrahamCampbell) [inline README.md]: Missing comma.

File: README.md [hunk 11]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+* CHeck the response : ```$response->isSent();```
+
+### Example
+Here is an exemple of a notification with Slack
+
+```PHP
+<?php
+
+// include autoloader
+include __DIR__ . '/vendor/autoload.php';
```
Comment (GrahamCampbell) [inline README.md]: This step can be skipped.

File: README.md (hunk 5 above)
Comment (Grummfy) [inline README.md:30]: Where do you want to add a new line? After "Usage"?

File: README.md (hunk 9 above)
Comment (Grummfy) [inline README.md]: How many space do you want?

File: README.md (hunk 10 above)
Comment (Grummfy) [inline README.md]: Why adding one? There is no need for it.

File: README.md (hunk 8 above)
Comment (Grummfy) [inline README.md:66]: You want all code without empty line?

File: README.md (hunk 5 above)
Comment (GrahamCampbell) [inline README.md:30]: Yes.

File: README.md (hunk 8 above)
Comment (GrahamCampbell) [inline README.md:66]: No. You've put way too many new lines after here. Like 6...

**Comment by Grummfy** [comment]
Did I still need to change something?

File: README.md [hunk 12]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
 * [Laravel 4](https://github.com/notifymehq/laravel4)
 * [Laravel 5](https://github.com/notifymehq/laravel5)
 
+## Usage
+
+* Create a factory : ```$factory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();```
```
Comment (GrahamCampbell) [inline README.md:30]: These are still wrong. Only a single backtick should be used.

File: README.md [hunk 13]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+* Make a notifier : ```$notifier = $factory->make([.]);```
```
Comment (GrahamCampbell) [inline README.md:31]: Don't put a dot in the array. It's not valid syntax.

File: README.md [hunk 14]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+
+// create a factory for notifications
+$notifierFactory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();
+
+// create the new notification for slack
+$slackNotifier = $notifierFactory->make([
+  // specify that we will use slack
+  'driver'  => 'slack',
+  // add api token to get access to slack API
+  'token'   => ''
```
Comment (GrahamCampbell) [inline README.md:53]: Still a missing comma here!

File: README.md [hunk 15]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+]);
+
+/* @var \NotifyMeHQ\NotifyMe\Response $response */
+$response =  $slackNotifier->notify('test message', [
+  // Who send this message, here is a bot called 'Super Bot'
+  'from' => 'Super Bot',
+  // Set the channel to received the message
+  'to'   => '#sandbox'
```
Comment (GrahamCampbell) [inline README.md:61]: Missing comma.

File: README.md [hunk 16]
```diff
@@ -25,6 +25,45 @@ Supported Bridges:
...
+// include autoloader if needed
+// include __DIR__ . '/vendor/autoload.php';
+
+// create a factory for notifications
+$notifierFactory = new NotifyMeHQ\NotifyMe\NotifyMeFactory();
+
+// create the new notification for slack
+$slackNotifier = $notifierFactory->make([
+  // specify that we will use slack
+  'driver'  => 'slack',
```
Comment (GrahamCampbell) [inline README.md:51]: Still too indented.

**Comment by GrahamCampbell** [comment]
cba with arguing. I'll just fix it myself. Thanks though.

**Pull request merged**